ansible/rename-user-on-old-nodes.yml

```diff
@@ -23,51 +22,55 @@
         fail_key: False
       register: old_user_present
 
+    - name: Check if epoch group exists
+      getent:
+        database: group
+        key: "{{ project_old_user }}"
+        split: ':'
+        fail_key: False
+      register: old_group_present
+
     - name: Check aeternity binary
       stat:
-        path: "{{ project_root }}/bin/epoch"
+        path: "{{ project_root }}/bin/aeternity"
       register: ae_node
 
     - name: Ping aeternity node
-      command: "{{ project_root }}/bin/epoch ping"
+      command: "{{ project_root }}/bin/aeternity ping"
       failed_when: no
       register: ping
       when: ae_node.stat.exists == True
 
     - name: Stop aeternity node
-      command: "{{ project_root }}/bin/epoch stop"
-      when: (ae_node.stat.exists == True and ping.stdout == "pong") or
+      command: "{{ project_root }}/bin/aeternity stop"
+      register: stopped
+      when: ae_node.stat.exists == True and ping.stdout == "pong" and
             old_user_present.ansible_facts.getent_passwd.epoch != None
 
-    - name: Check new home directory
-      stat:
-        path: "{{ project_new_home }}"
-      register: new_home_present
+    - name: Find epmd binary
+      find:
+        paths:
+          - "{{project_old_home}}"
+          - "{{ project_new_home }}"
```

Comment:
one var have spaces one do not have ;] just looks not nice ;]

Reply:
Fixed in 4a42c1f.

Reply:
@valkov-ivaylo yea... can you make this consistent for whole playbook? :D it looks much beter with spaces, and we use spaces everywhere {{ something }}